packages/cli/src/commands/verify.ts

```diff
@@ -21,6 +21,47 @@ export default class Verify extends Command {
       min: 0,
       default: 1,
     }),
+    'certificate-issuer': Flags.string({
+      description:
+        "Value that must appear in the signing certificate's issuer extension (OID 1.3.6.1.4.1.57264.1.1). Not verified if no value is supplied",
+    }),
+    'certificate-identity-email': Flags.string({
+      description:
+        "Email address which must appear in the signing certificate's Subject Alternative Name (SAN) extension. Not verified if no value is supplied",
+      dependsOn: ['certificate-issuer'],
+      exclusive: ['certificate-identity-uri'],
+    }),
+    'certificate-identity-uri': Flags.string({
+      description:
+        "URI which must appear in the signing certificate's Subject Alternative Name (SAN) extension. Not verified if no value is supplied",
+      dependsOn: ['certificate-issuer'],
+      exclusive: ['certificate-identity-email'],
+    }),
+    'certificate-oids': Flags.string({
+      description:
+        "A collection of OID/value pairs which must be present in the certificate's extension list. Not verified if no value is supplied",
+    }),
```

Comment:
Let's ditch this flag for now. I noticed that it wasn't passed-in to the verifier and it's not implemented yet even if we were to pass it in.

Reply:
Ah yea, forgot about this one! I'll remove it!